scripts/workers/code_upload_worker_utils/make_submission.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+echo "File Submission Started."
+submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/challenge_phase/$PHASE_PK/submission/$SUBMISSION_PK"
+submission_curl_base_request="curl --location --request PATCH '$submission_url' --header 'Authorization: Bearer $AUTH_TOKEN'"
+if [ -f "$SUBMISSION_PATH/submission.csv" ]
```

Comment:
We need to take this extension as part of challenge config. For now we can use this script. Once we test everything we can switch to accepting submisison extension from challenge config